tests/copp/test_copp.py

```diff
@@ -85,6 +85,17 @@ def test_no_policer(self, protocol, duthost, ptfhost, copp_testbed):
             Checks that the policer does not enforce a rate limit for protocols
             that do not have any set rate limit.
         """
+        # FIXME: The DHCP COPP Policy was removed from T1 in a recent change, so no
```

Comment:
Should we have it as a separate PR since this is related to dhcp and title is different? or suggest change title.

Reply:
That's a good point, I can update the title to something more generic about COPP fixes! @yxieca do you have a preference?

Reply:
Updated the PR info, let me know if I should split it up.

Reply:
@daall I have some changes in internal branch for DHCP Related COPP. We can remove this check when my PR is ready.

Reply:
I talked to @prsunny yesterday and it sounded like DHCP packets wouldn't be trapped at all on T1. Do the internal changes change that behavior?

Reply:
@daall I made the change in ansible/roles/test/files/ptftests/copp_tests.py
to check rx_pps and packet count to be 0 fro DHCP so test case will pass. 
I need to still qualify that change with testbed-type (T0 vs T1)

Reply:
Oh sweet! Thanks. :)